thundra/integrations/botocore.py

```diff
@@ -135,49 +119,44 @@ def get_operation_name(self, wrapped, instance, args, kwargs):
         return str(self.getQueueName(request_data))
 
     def getRequestType(self, string):
-        if string in Constants.SQSRequestTypes:
-            return Constants.SQSRequestTypes[string]
+        if string in constants.SQSRequestTypes:
+            return constants.SQSRequestTypes[string]
         return 'READ'
 
     def getQueueName(self, data):
         if 'QueueUrl' in data:
             return data['QueueUrl'].split('/')[-1]
         elif 'QueueName' in data:
             return data['QueueName']
-        return Constants.AWS_SERVICE_REQUEST
-
-    def inject_span_info(self, scope, wrapped, instance, args, kwargs, response,
-                         exception):
+        return constants.AWS_SERVICE_REQUEST
```

Comment:
return empty string instead of `AWS_SERVICE_REQUEST` constant if the queue name is not available

Reply:
But be sure that in case of empty queue name, operation name is not empy but `AWS_SERVICE_REQUEST` constant